pypeit/datamodel.py

```diff
@@ -1143,7 +1199,19 @@ def to_hdu(self, hdr=None, add_primary=False, primary_hdr=None,
                 ext = list(d.keys())[0]
                 # Allow for embedded DataContainer's
                 if isinstance(d[ext], DataContainer):
-                    hdu += d[ext].to_hdu(add_primary=False)
+                    # TODO: This is a hack because I want to be able to
+                    # override the extension names of DataContainers
+                    # that are confined to a single HDU. At the moment,
+                    # this allows for the LEFT_PCA and RIGHT_PCA
+                    # extensions (both TracePCA DataContainers) in the
+                    # MasterEdge output file. We might be able to do
+                    # this with `hdu_prefix` if we made it an instance
```

Comment:
I played with this and realized that it has to be a class attribute so that it can be called by things like `_parse`.  I made some slight changes to the code and edited this comment to reflect that.